scripts/jenkins_regress.sh

```diff
@@ -24,10 +24,10 @@ version=`git rev-parse HEAD`
 
 if [ -z $pr_code_coverage ];
  then
-        echo "For master"
+        echo "For main"
 	/usr/local/sonar-scanner-cli/bin/sonar-scanner \
-    	-Dsonar.host.url=https://sonarqube.int.snowflakecomputing.com \
-    	-Dsonar.projectBaseDir=${PWD} \
+    	-Dsonar.host.url=https://sonarqube-eng.int.snowflakecomputing.com \
+    	-Dsonar.projectBaseDir=/mnt/jenkins/home/jenkins/workspace/SnowparkClientRegressRunner/thundersnow \
```

Comment:
NIT: It should be better to find a relative path from Jenkins work dir, e.g. `$<jenkins_work_dir>/SnowparkClientRegressRunner/thundersnow`, though I am not sure what it is.